app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -73,14 +112,28 @@ async def pixetl_symbology(
             "calc": None,
             "resampling": PIXETL_DEFAULT_RESAMPLING,
             "grid": f"zoom_{zoom_level}",
-            "pixel_meaning": pixel_meaning,
         },
     )
+
+    # Needed by gdaldem down the line
+    source_asset_uri = tile_uri_to_tiles_geojson(
+        get_asset_uri(
+            dataset,
+            version,
+            AssetType.raster_tile_set,
+            creation_options.dict(by_alias=True),
+            "epsg:3857",
+        )
+    )
+
+    pixel_meaning = f"{source_asset_co.pixel_meaning}_{source_asset_co.symbology.type}"
+    creation_options.pixel_meaning = pixel_meaning
```

Comment:
move above line 108 and update creation_options only once.
You should set `source_uri` attribute to `source_asset_uri`. This way the correct value will be saved in the creation options and you don't need to pass around an extra parameter.